tests/e2e/testRunner.js

```diff
@@ -203,8 +203,8 @@ const runTests = async () => {
             }
         }
 
-        const coolDownLogs = Logger.progressInfo(`Cooling down for ${config.COOL_DOWN / 1000}s`);
-        coolDownLogs.updateText(`Cooling down for ${config.COOL_DOWN / 1000}s`);
+        const coolDownLogs = Logger.progressInfo(`Cooling down for ${config.BOOT_COOL_DOWN / 1000}s`);
+        coolDownLogs.updateText(`Cooling down for ${config.BOOT_COOL_DOWN / 1000}s`);
```

Comment:
Updated, because `COOL_DOWN` value [was removed](https://github.com/Expensify/App/pull/30635/files#diff-f574ede8492c922c6594dac2e8a92624c1b464cf4fdfa9202f0b5de595b33478L50) from config, and it was showing `Cooling down for NaNs` message. These logs are related to `BOOT_COOL_DOWN`, you can take a look at [merged PR](https://github.com/Expensify/App/pull/30635/files#diff-b139fe8c7ddb7b849942557484a688be889f47cadee4ec0d4d2c9d8bb9d2fcf4R212) for more context